test/vhswish-microkernel-tester.h

```diff
@@ -170,8 +170,8 @@ class VHSwishMicrokernelTester {
         const int32_t input_value = int32_t(uint32_t(input_zero_point() - input[i]) << 7);
         int32_t in = input_value * input_scale_div;
         in -= 16384;  // subtract 0.5 in Q15
-        in = std::min(in, 0);
-        in = std::max(in, -32768);
+        in = std::min<uint32_t>(in, 0);
+        in = std::max<uint32_t>(in, -32768);
```

Comment:
Same comment here as above

Reply:
This also makes sense to take negative integers, so int32_t is the correct one to add. I will recommit this as well, thank you!